packages/mui-docs/src/MarkdownElement/MarkdownElement.tsx

```diff
@@ -559,6 +559,9 @@ const Root = styled('div')(
       transition: theme.transitions.create(['background', 'borderColor', 'display'], {
         duration: theme.transitions.duration.shortest,
       }),
+      '@media (max-width: 640px)': {
```

Comment:
Why does it use a custom value? Meaning, why is this not using the theme breakpoints?